### ISSUE TITLE
Installing Encoder is not working

### ISSUE THREAD

**Issue by dbmasters** [WWBN/AVideo#3503]
I have aVideo installed, it works fine, it keeps wanting to use the network encoder, I want my own encoder installed, so I downloaded it, uploaded it into a root directory named "encoder" and tried to run thru the install...it keeps giving me a generic "unknown error" type of message, in looking thru the code that is thrown at the very end of the script in case it didn't work and wasn't already caught...

I have got it installed once before while going thru various iterations of this project, but only once, using this same method each time. Install documentation is not very informative, unless I missed something in the read me or other documentation.

Anyone have any advice?

Should the site and path in the install be set to the encoder directory or the AVideo directory? Should encoder be a subdomain and in it's own database? Seems like maybe it should be...I was just diggin thru the SQL script and it seems they both have configurations tables...

**Comment by dbmasters** [comment]
OK, I got this installed...

Just FYI, I gave encoder it's own database, left it in the subdirectory, it works...annoyed me at first, but is kind of clever in a way, the encoder could be on a bigger server individually, so I get that, but installation did take a lot of trial/error...would be nice to have some of that documented more clearly.

Also, the encoder isn't very intuitive...it uploads behind the scenes...no submit button or anything, so while cool and automatic I did have some immediate feedback regarding the UI/UX of the encoder. Just submitting that for feedback to the maker.

Installing the streamer was easy, encoder could use some better instruction...my business partner and I documented our experience pretty well for future reference.
